SilKit/source/core/participant/Participant_impl.hpp

```diff
@@ -721,8 +722,8 @@ auto Participant<SilKitConnectionT>::CreateTimeSyncService(Orchestration::Lifecy
     config.name = Discovery::controllerTypeTimeSyncService;
     config.network = "default";
     timeSyncService = CreateController<Orchestration::TimeSyncService>(
-        config, std::move(timeSyncSupplementalData), false, &_timeProvider, _participantConfig.healthCheck,
-        lifecycleService);
+        config, std::move(timeSyncSupplementalData), false, false, &_timeProvider, _participantConfig.healthCheck,
+        lifecycleService, _participantConfig.experimental.timeSynchronization.animationFactor);
```

Comment:
All services that can be created using the `CreateController` function tempalte get passed a `IParticipantInternal *` as the first argument. The configuration can be retrieved by the `TimeSyncService` constructor from the participant directly.